src/web/src/components/Posts/GitHubInfo.tsx

```diff
@@ -2,6 +2,7 @@ import { createStyles, makeStyles, Theme, ListSubheader } from '@material-ui/cor
 import Repos from './Repos';
 import Issues from './Issues';
 import PullRequests from './PullRequests';
+import { Url } from 'url';
```

Comment:
This isn't needed.  `URL` is [part of the web platform](https://caniuse.com/url)